app/models/StatusCheckResult.scala

```diff
@@ -14,16 +14,15 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.homeofficesettledstatusstubs.models
+package models
 
-import play.api.libs.json.{Format, Json}
+import play.api.libs.json._
 
 import java.time.LocalDate
 
 final case class StatusCheckResult(
   fullName: String,
   dateOfBirth: LocalDate,
-  // (ICAO 3 letter acronym - ISO 3166-1)
```

Comment:
Yh I removed this because the message is clear on line 29 and the example JSONs added should used the the nationality DEU for the example JSON. Don't feel the need to add it several places as it should evident I think.